async_downloads/cache.py

```diff
@@ -108,6 +108,8 @@ def save_download(download_key, iterable=None, file=None):
             download["errors"] = str(e)
     download["complete"] = True
     download["percentage"] = 100
+    if download["complete"]:
```

Comment:
Do we need the `if` statement as the dictionary value was set in line 109?

Reply:
we do not :smiley: